ios/ConvosNSE/SentryManager.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Sentry
+
+final class SentryManager {
+  static let shared = SentryManager()
+
+  private var sentryInitialized = false
+
+  private init() {}
+
+  func initSentry() {
+    DispatchQueue.main.async {
+      guard !self.sentryInitialized else { return }
+
+      let sentryEnv = try! getInfoPlistValue(key: "Env", defaultValue: "dev")
+
+      SentrySDK.start { options in
+        options.dsn = "https://o4504757119680512.ingest.us.sentry.io/4509079521067008"
```

Comment:
DSN should be `https://dc072f632e53ca5d87b47120ad0a2e31@o4504757119680512.ingest.us.sentry.io/4509079521067008`